main.go

```diff
@@ -3,12 +3,47 @@ package main
 import (
 	"net/http"
 
-	"github.com/gorilla/mux"
+	"github.com/DataDog/datadog-go/v5/statsd"
+
+	"os"
+
+	log "github.com/sirupsen/logrus"
+
+	muxtrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+var logFile = "/tmp/instrumentedhttpserver.log"
+
 func main() {
-	r := mux.NewRouter()
-	todoserver := NewServer()
+
+	// logs
+	f, err := os.OpenFile(logFile, os.O_APPEND|os.O_CREATE|os.O_RDWR, 0666)
+	if err != nil {
+		// log.Fatal(err)
+	}
+	defer f.Close()
+
+	log.SetFormatter(&log.JSONFormatter{})
+	log.SetOutput(f)
+	log.WithFields(log.Fields{"string": "foo", "int": 1, "float": 1.1}).Info("instrumented http server started...")
+
+	// statsd
+	statsd, err := statsd.New("127.0.0.1:8125")
```

Comment:
fine for now, but as we discussed offline, this will need to change for a containerized environment